src/VueTyping.spec.js

```diff
@@ -1,82 +1,90 @@
-import { createLocalVue, mount } from '@vue/test-utils'
-import VueTyping from '.'
+import { createLocalVue, mount } from "@vue/test-utils";
```

Comment:
The files formatting is different. I've recently added eslint, so if you run `yarn lint --fix` you should be able to get files in the new formatting

Reply:
I will fix it !